projects/angular/src/forms/styles/_checkbox.clarity.scss

```diff
@@ -106,8 +112,8 @@
       border-left: none;
       @include mixins.css-var(
         border-bottom-color,
-        $indetermCssBrdrColorVar,
-        $indetermSassBrdrColorVar,
+        clr-forms-checkbox-indeterminate-border-color,
+        forms-variables.$clr-forms-checkbox-indeterminate-border-color,
```

Comment:
Looks like both of these are `indeterminate` so they should use the same variable